storage/volume/types/volume.go

```diff
@@ -193,21 +200,25 @@ func (v *Volume) CreateTime() string {
 
 // VolumeID use to define the volume's identity.
 type VolumeID struct {
```

Comment:
Could we change the name of VolumeID？It is so strange.
Is this that this VolumeID will be stored in some legacy db?